documentation/docs/pages/operators/binaries/building-nym.mdx

```diff
@@ -54,17 +64,17 @@ cargo build --release # build your binaries with **mainnet** configuration
 
 Quite a bit of stuff gets built. The key working parts are:
 
-* [Nym Node](../nodes/nym-node.md): `nym-node`
-* [Validator](../nodes/validator-setup.md)
+* [Nym Node](../nodes/nym-node.mdx): `nym-node`
+* [Validator](../nodes/validator-setup.mdx)
 * [websocket client](https://nymtech.net/developers/clients/websocket-client.html): `nym-client`
```

Comment:
these links need changing

Reply:
changed the top one, validator one is good.

all of the ones pointing to developers or networks or sdks are to be done post merge. 

I will grep them